python_modules/libraries/dagster-aws/dagster_aws/ecs/container_context.py

```diff
@@ -53,6 +53,16 @@
             }
         )
     ),
+    "run_ecs_tags": Field(
+        Array(
+            {
+                "key": Field(StringSource, is_required=True),
+                "value": Field(StringSource, is_required=False),
+            }
+        ),
+        is_required=False,
+        description="Additional tags to apply to the launched ECS task.",
+    ),
```

Comment:
Putting this in SHARED_ECS_SCHEMA ensures that this field is available to set at both instance-level config (applying to all runs) and also all runs for a particular code location via EcsContainerContext

When a run is created (at enqueue time), Dagster attaches the container context from the code location to the run. When the run is launched, that container context is pulled off the run, merged with anything set at the instance level, and used to create the actual container/task/pod/whatever